chia/cmds/wallet_funcs.py

```diff
@@ -47,15 +50,22 @@ async def get_transactions(args: dict, wallet_client: WalletRpcClient, fingerpri
     paginate = args["paginate"]
     if paginate is None:
         paginate = sys.stdout.isatty()
-    txs: List[TransactionRecord] = await wallet_client.get_transactions(wallet_id)
+
+    starttx = args.get("offset", 0)
```

Comment:
although, I notice you updated to documentation to say "recent", which would suggest that 0 is the most recent, which makes it impossible (afaict) to reliably iterate over all transactions in multiple requests

Reply:
perhaps it's overkill here; but I think the most reliable API to support this would be to return a cookie to be passed back in the next call, to resume the iteration. That way the node can encode the cursor (however it likes to encode it) in the cookie, and resume from where it last left off. It disconnects the cursor from what the first item happens to be.

Reply:
I added the ability to sort either newest first or oldest first. Using oldest first, the CLI will reasonably paginate - however, the implementation is workable but not ideal.